backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/backup/StopVmBackupCommand.java

```diff
@@ -94,7 +98,11 @@ protected void executeCommand() {
                 }
             } else {
                 log.error("Failed to stop backup '{}' in '{}' phase", vmBackup.getId(), phase);
-                updateVmBackupPhase(VmBackupPhase.FINALIZING_FAILURE);
+                if (isHybridBackup()) {
+                    updateVmBackupStopped();
```

Comment:
we cannot get here with hybrid backups because stopVmBackup always returns true (which is confusing, we better change that code to make it more explicit)

Reply:
stopVmBackup() returns false if StopVmBackup VDS command fails. Cannot it happen?

Reply:
the VDS command is only relevant for live backup which doesn't run for hybrid backup

Reply:
Good point.

Reply:
Done